job-server/test/spark.jobserver/LocalContextSupervisorSpec.scala

```diff
@@ -28,9 +25,14 @@ object LocalContextSupervisorSpec {
           memory-per-node = 512m
         }
       }
+
       context-settings {
         num-cpu-cores = 2
         memory-per-node = 512m
+        passthrough {
+          spark.driver.allowMultipleContexts = true
```

Comment:
Hm. Could we add this `allowMultipleContexts` setting to application.conf as well?  I mean it seems to be good default for job server.